### PR TITLE
Catalog scraper updates

### PR DESCRIPTION
**Pull request by imanmoreira**
Updating the catalog scraper to include Concentrations and new Course blocks for subsections that require specific credit requirements.

### CODE REVIEW

File: scrapers/src/catalog-scraper/row_scraper.ts [hunk 1]
```diff
@@ -158,10 +195,15 @@ export function parseSubjectRangeRow(
   row: CheerioElement
 ): Array<ISubjectRange> {
   let currentRow: Cheerio = $(row);
-  let anchors: Cheerio = currentRow.find(".courselistcomment.commentindent");
-
-  if (anchors.length == 0) {
-    return parseAsFullRange($, row);
+  let anchors: Cheerio = currentRow.find(".courselistcomment");
+  // If the anchor isn't indented as a comment or contains "course" or "elevtive" it could be a full range
```
Comment (ArunJ98) [inline scrapers/src/catalog-scraper/row_scraper.ts:199]: nit - elective spelled wrong here

File: scrapers/src/catalog-scraper/reqGroup_scraper.ts [hunk 2]
```diff
@@ -202,6 +219,118 @@ function processAndSection(
...
+  let split = new Map<CheerioElement[], [SubSectionType, number]>();
+  let sectionType = SubSectionType.COURSES;
+  let credits = 0;
+  let rowCollection: CheerioElement[] = [];
+  rows.forEach((row: CheerioElement) => {
+    let comment = $(row)
+      .find("span.courselistcomment")
+      .text();
+    if (comment.length > 0) {
+      if (ORTagMap.hasOwnProperty(comment)) {
```
Comment (ArunJ98) [inline scrapers/src/catalog-scraper/reqGroup_scraper.ts:293]: The nested ifs could be helper methods, that would make this function a lot more readable

**Review by ArunJ98 — DISMISSED** [review]
Overall looks pretty good to me, left a couple small comments but other than that, you have a lot of merge conflicts

File: scrapers/src/catalog-scraper/reqGroup_scraper.ts (hunk 2 above)
Comment (imanmoreira) [reply scrapers/src/catalog-scraper/reqGroup_scraper.ts:293]: I tried it out and it required a function with 7 arguments, which seemed like too much to make it worth it

File: package.json [hunk 3]
```diff
@@ -8,15 +8,16 @@
   "workspaces": [
     "frontend",
     "scrapers",
-    "api"
+    "api",
+    "graduate-common"
```
Comment (ArunJ98) [inline package.json:12]: this should just be common instead of graduate-common

**Review by ArunJ98 — COMMENTED** [review]
one minor comment, but other than that, looks good

**Review by NilsBacke — APPROVED** [review]

**Pull request merged**